AdminWebsite/AdminWebsite/ClientApp/src/app/services/date-and-time.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class DateAndTimeService {
+    minutesToHours(totalMinutes: number): { hours: number; minutes: number } {
+        const hours = Math.floor(totalMinutes / 60);
+        const minutes = totalMinutes % 60;
+        return { hours, minutes };
+    }
+
+    minutesToHoursDisplay(totalMinutes: number): string {
+        const time = this.minutesToHours(totalMinutes);
+        const hours = `${time.hours}${time.hours > 1 ? 'hrs' : 'hr'}`;
+        const minutes = time.minutes !== 0 ? ` ${time.minutes}min` : '';
+        return hours + minutes;
```

Comment:
not a deal breaker but it might be an easier read to not worry about white spaces in the hours and minutes in the last statement
```suggestion
        return `${hours} ${minutes}`;
```

Reply:
I considered doing it like that but then thought, it's will have a trailing space if the minutes are 0 and not included in the return string, because of the space between hours and minutes. (Can return